LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/authentication/IAccessTokenAccessor.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.labapi.utilities.authentication;
+
+import com.microsoft.identity.labapi.utilities.authentication.exception.LabApiException;
+
+/**
+ * An interface describing an access token accessor i.e. anyone that has the ability to return an
+ * access token.
+ */
+public interface IAccessTokenAccessor {
+
+    /**
+     * Obtain an access token.
+     *
+     * @return a String representing an access token
+     */
+    String getAccessToken() throws LabApiException;
```

Comment:
Probably would be nice to go ahead and give this a type with one field, if we're really making a new Interface.  That way it's clear what's in the String.

Reply:
OK, that's kind what's happening?  It's just using an interface instead of a class.

Reply:
Well technically anyone can choose to implement this interface and choose to put garbage into the string

Reply:
That's fine, but the comment is really about what you do with the resulting value. It's easier if there's type enforcement. Not that important.